web/client/components/map/openlayers/PopupSupport.jsx

```diff
@@ -59,6 +61,14 @@ export default class PopupSupport extends React.Component {
         this.props.onPopupClose(id);
     }
     renderPopups = () => {
+        const {identifyPopupHidden, map} = this.props;
+        if (identifyPopupHidden) {
```

Comment:
maybe the flag hidePopupIfNoResults  could have been used to filter out elements?
i have to debug this tbh